cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetServiceAuthorizationTest.java

```diff
@@ -291,12 +329,11 @@ public void execute() throws Exception {
     grantAndAssertSuccess(NamespaceId.DEFAULT, BOB, EnumSet.allOf(Action.class));
 
     // delete all instances so modules can be deleted
-    dsFramework.deleteAllInstances(NamespaceId.DEFAULT);
     SecurityRequestContext.setUserId(ALICE.getName());
     dsFramework.deleteAllInstances(NamespaceId.DEFAULT);
     SecurityRequestContext.setUserId(BOB.getName());
     // After granting admin on the default namespace, deleting all modules should succeed
-    grantAndAssertSuccess(NamespaceId.DEFAULT, BOB, EnumSet.of(Action.ADMIN));
+    grantAndAssertSuccess(module1, BOB, EnumSet.of(Action.ADMIN));
     dsFramework.deleteAllModules(NamespaceId.DEFAULT);
```

Comment:
No. Since BOB does not have privilege on module1, so he will not pass the privilege check for module1, which leads to `UnAuthorized Exception`